corehq/sql_db/tests/test_routers.py

```diff
@@ -1,9 +1,12 @@
+import mock as mock
```

Comment:
I don't understand what this line does that `import mock` wouldn't. Also `patch` was already imported a couple of lines down so I think you could have just used that.

Reply:
no idea why that's like that, removed: 2717285c800